tests/test_libraries.py

```diff
@@ -2623,3 +2623,11 @@ def test_pypdfium2(pyi_builder):
     pyi_builder.test_source("""
         import pypdfium2
     """)
+
```

Comment:
The functions need to be separated by two empty lines to appease the linter.

Reply:
Also fixed, thanks for the review!